pkg/dns/utils/utils.go

```diff
@@ -83,3 +86,7 @@ func (this *LogMessage) Warnf(logger logger.LogContext, add ...interface{}) bool
 func (this *LogMessage) Debugf(logger logger.LogContext, add ...interface{}) bool {
 	return this.out(logger.Debugf, add...)
 }
+
+func SetLastUpdateTime(lastUptimeTime **metav1.Time) {
```

Comment:
Do you really need `**metav1.Time`?
With `*metav1.Time` you are already capable of manipulating the content of the address space `lastUptimeTime` points to.
Can you also add documentation?

Reply:
Yes, the double pointer is needed. Initially the `*metav1.Time` is nil and you need to assign the address of the struct.
Regarding documentation: if you call `kubectl get crd dnsentries` you get a description for all status fields including the new field `lastUpdateTime`.

Reply:
Okay, thanks. I missed the `nil` case.
> Regarding documentation: if you call kubectl get crd dnsentries you get a description for all status fields including the new field lastUpdateTime.
This was related to the function. Having an inline comment for exported elements is considered best practice in Go, [ref](https://golang.org/doc/effective_go.html#commentary).

Reply:
Ah ok. I also added some documentation for the status of `DNSProvider` and `DNSEntry`. See `/docs/usage`